I'm getting the same error on Ubuntu as on the CI.
```
======================================================================
FAIL: test_make_frame (idlelib.idle_test.test_searchbase.SearchDialogBaseTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/cheryl/cpython/Lib/idlelib/idle_test/test_searchbase.py", line 101, in test_make_frame
    self.assertIsInstance(frame, Frame)
AssertionError: <tkinter.ttk.Frame object .!frame7> is not an instance of <class 'tkinter.ttk.Frame'>

----------------------------------------------------------------------

```
If you want me to look into in more detail, let me know.

--- a/Lib/idlelib/idle_test/test_searchbase.py
+++ b/Lib/idlelib/idle_test/test_searchbase.py
@@ -101,7 +101,7 @@ class SearchDialogBaseTest(unittest.TestCase):
         self.assertEqual(str(type(frame)), "<class 'tkinter.ttk.Frame'>")
         # self.assertIsInstance(frame, Frame) fails when test is run by
         # test_idle not run from IDLE editor.  See issue 33987 PR.
- 
+
         frame, label = self.dialog.make_frame('testlabel')
         self.assertEqual(label['text'], 'testlabel')
 

--- a/Lib/idlelib/idle_test/test_searchbase.py
+++ b/Lib/idlelib/idle_test/test_searchbase.py
@@ -98,11 +98,12 @@ class SearchDialogBaseTest(unittest.TestCase):
         self.dialog.top = self.root
         frame, label = self.dialog.make_frame()
         self.assertEqual(label, '')
-        self.assertIsInstance(frame, Frame)
-
+        self.assertEqual(str(type(frame)), "<class 'tkinter.ttk.Frame'>")
+        # self.assertIsInstance(frame, Frame) fails when test is run by
+        # test_idle not run from IDLE editor.  See issue 33987 PR.
+ 
         frame, label = self.dialog.make_frame('testlabel')
         self.assertEqual(label['text'], 'testlabel')
-        self.assertIsInstance(frame, Frame)
 
     def btn_test_setup(self, meth):
         self.dialog.top = self.root

